PennyPincherExample/PennyPincherExample/ViewController.swift

```diff
@@ -52,8 +49,61 @@ class ViewController: UIViewController, UITextFieldDelegate {
         recognizerResultLabel.text = "Template: \(template.id), Similarity: \(similarityString)"
     }
    
+    //MARK:- Actions
+    @IBAction func didTapAddTemplate(_ sender: AnyObject) {
+        if
+            let text = templateTextField.text,
+            let template = PennyPincher.createTemplate(text, points: gestureView.points) {
```

Comment:
Not to step on the toes of this project's style, but consider either moving the opening brace to the next line or leaving a blank line after this one. At a glance it doesn't look like the beginning of the block statement, it looks like more conditional.